src/main/java/teammates/common/datatransfer/attributes/InstructorAttributes.java

```diff
@@ -18,131 +18,60 @@
     public static final String DEFAULT_DISPLAY_NAME = "Instructor";
 
     // Note: be careful when changing these variables as their names are used in *.json files.
+
+    /** Required fields. */
     public String googleId;
     public String courseId;
     public String name;
     public String email;
-    public Boolean isArchived;
+
+    /** Optional fields. */
     public String key;
     public String role;
-    public boolean isDisplayedToStudents;
     public String displayedName;
-
+    public Boolean isArchived;
```

Comment:
If Boolean is used in Instructor just for storage, it doesn't make sense to have Boolean object for datatransfer (see `isDisplayedToStudents` is using primitive type). Maybe we could have an issue on this? @whipermr5